src/main/java/com/snowflake/kafka/connector/streaming/iceberg/IcebergDDLTypes.java

```diff
@@ -0,0 +1,49 @@
+package com.snowflake.kafka.connector.streaming.iceberg;
+
+public class IcebergDDLTypes {
+  public static String INTEGER = "INTEGER";
+  public static String STRING = "STRING";
+  public static String BIGINT = "BIGINT";
+  public static String MAP = "MAP";
+  public static String OBJECT = "OBJECT";
+  public static String VARCHAR = "VARCHAR";
+
+  public static String ICEBERG_METADATA_OBJECT_SCHEMA =
```

Comment:
nit: The formatting is so bad that I would consider removing const Strings to achieve better readability.

Reply:
yea, agree, didn't have a look at it after using formatter, super ugly